frontend/src/user/categories/WordItemResult.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { RootState } from '@store/store';
+import { Word } from '@store/words';
+import { CheckIcon, CrossIcon } from '@icons/';
+
+export default function WordItemResult({ word }: { word: Word }) {
+  const state = useSelector((state: RootState) => state);
+  const answer = state.words.answers.find((choiceAnswer) => {
+    return choiceAnswer.word_id === word.id;
+  });
+  const correctAnswer = word.choices.find((choice) => {
+    return choice.is_correct
+  });
+
+  return (
+    <div className='word-result-group'>
+      <div className='word-column'>
+        <span className='italic'>word</span>
+        <span
+          key={word.id}
+          className='md:text-4xl xs:text-2xl text-purple-400 font-bold'
+        >
+          {word.word}
+        </span>
+      </div>
+      <div className='choices-row'>
+        <div className='flex w-3/4 justify-evenly'>
+          {word.choices.map((choice, index) => {
+            return (
+              <>
+                <div className='flex flex-col w-1/5 items-center'>
+                  <span
+                    key={choice.id}
+                    className={`choice-item-result
+                      ${choice.is_correct && 'bg-success'}
+                      ${
+                        answer?.id === choice.id &&
+                        !choice.is_correct &&
+                        'bg-failed'
+                      }
+                    `}
+                  >
+                    {choice.name}
+                  </span>
+                  {answer?.id === choice.id && (
```

Comment:
remove the `0` under the word
![image](https://user-images.githubusercontent.com/20507565/150281303-166d7f18-3514-44ea-97b2-9615ce18f7f9.png)